inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/file/FileSourceDTO.java

```diff
@@ -82,6 +82,9 @@ public class FileSourceDTO {
     @ApiModelProperty("Column separator of data source ")
     private String dataSeparator;
 
+    @ApiModelProperty("Metadata filters by label")
+    private Map<String, String> filterMetaByLabels;
```

Comment:
Why is this field in `FileSource` is `List<Map<>>`, but here is `Map<>`?

Reply:
done